<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QWidget" name="Viewer">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*  ~COLOR SCHEME~
Light Background = #757575
Dark Background = #454545
Borders and Dark Text = #000000
Light Text = #FFFFFF
Highlight Color = #4090EE
Debug Color = #22EE0E
Info Color = #FFFFFF
Warning Color = #4090EE
Error Color = #F00000
*/

QWidget {
	background-color: #757575;
	color: #000000;
	font: 11pt &quot;Monospace&quot;;
}

QTextBrowser {
	color: #FFFFFF;
	background-color: #454545;
	border: 1px solid #000000;
}

QLineEdit {
	color: #FFFFFF;
	background-color: #454545;
	border: 1px solid #000000;
}

QTabWidget::pane { /* The tab widget frame */
    border: 1px solid #000000;
}
QTabBar::tab {
	padding: 3px 15px;
	margin: 0px 9px 0px 0px;
    border: 1px solid #000000;
    border-bottom: 2px solid #454545
}
QTabBar::tab:selected {
    border-bottom-color: #4090EE;

}

QPushButton {
	color: #FFFFFF;
	background-color: #454545;
	border: 1px solid #000000;
	margin: 0px;
	padding: 10px;
}
QPushButton:open {
	color: #000000;
	background-color: #4090EE;
}
QPushButton#DButton:open {
	background-color: #22EE0E;
}
QPushButton#IButton:open {
	background-color: #FFFFFF;
}
QPushButton#WButton:open {
	background-color: #4090EE;
}
QPushButton#EButton:open {
	background-color: #F00000;
}
QPushButton#CloseButton {
	padding:0 2 0 2px;
}

QSlider::groove {
    border: 1px solid #151515;
    height: 2px;
    background: #757575;
    margin: 2px 0;
}
QSlider::handle {
    background: #454545;
    border: 1px solid #151515;
    width: 9px;
    margin: -6px 0;
    border-radius: 3px;
}
QSlider::add-page {
    border: 1px solid #151515;
    height: 2px;
    background: #4090EE;
    margin: 2px 0;
}
QSlider::sub-page {
    border: 1px solid #151515;
    height: 2px;
    background: #757575;
    margin: 2px 0;
}
QSlider#LateSlider::add-page {
    border: 1px solid #151515;
    height: 2px;
    background: #757575;
    margin: 2px 0;
}
QSlider#LateSlider::sub-page {
    border: 1px solid #151515;
    height: 2px;
    background: #4090EE;
    margin: 2px 0;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Log">
      <attribute name="title">
       <string>Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="View">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Monospace'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>No Messages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mouseover text for timestamp</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Options">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="RefreshButton">
           <property name="text">
            <string>Refresh Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Path">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>/var/log/trusthub.log</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="CutoffLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="LateLabel">
           <property name="text">
            <string>Late Cutoff</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSlider" name="EarlySlider">
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="EarlyLabel">
           <property name="text">
            <string>Early Cutoff</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="LateSlider">
           <property name="value">
            <number>99</number>
           </property>
           <property name="sliderPosition">
            <number>99</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="LevelLayout">
         <item>
          <widget class="QPushButton" name="DButton">
           <property name="text">
            <string>Debug Level</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="IButton">
           <property name="text">
            <string>Info Level</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="WButton">
           <property name="text">
            <string>Warning Level</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="EButton">
           <property name="text">
            <string>Error Level</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="PButton">
           <property name="text">
            <string>View Policy Engine Logs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="KButton">
           <property name="text">
            <string>View Kernel Module Logs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="CloseButton">
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
